ceph.spec.in

```diff
@@ -504,9 +536,29 @@ Requires:	libcephfs1 = %{epoch}:%{version}-%{release}
 Requires:	python-rados = %{epoch}:%{version}-%{release}
 Obsoletes:	python-ceph < %{epoch}:%{version}-%{release}
 %description -n python-cephfs
-This package contains Python libraries for interacting with Cephs distributed
+This package contains Python 2 libraries for interacting with Cephs distributed
 file system.
 
+%package -n python%{python3_pkgversion}-cephfs
+Summary:	Python libraries for Ceph distributed file system
```

Comment:
Python 2 libraries

(lots of these are left - you can find them by grepping for "Python libraries")